packages/static/src/progress-bar/progress-bar.tsx

```diff
@@ -0,0 +1,164 @@
+import clampValue from 'lodash/clamp'
+import React, { useEffect, useMemo } from 'react'
+import CssInjection from '../utils/objectToCss/CssInjection'
+import { useDOMRef } from '../utils/use-dom-ref'
+import styles from './styles/progress-bar.module.css'
+
+type Props = {
+  rightLabel?: string
+  /** @default "lg" */
+  size?: 'sm' | 'md' | 'lg' | 'xl'
+  /**
+   * Color visualization of the current value.
+   * @default "blue" // $info - #009EDA
+   */
+  color?: string
+  /**
+   * Color of the progress bar.
+   * @default "$gray30" // #EDEBE9
+   */
+  barColor?: string
+  /** @default "normal" */
+  variant?: 'normal' | 'rounded'
+  label?: string
+  /** @default 0 */
+  value?: number
+  /** @default 0 */
+  minValue?: number
+  /** @default 100 */
+  maxValue?: number
+  onComplete?: () => void
+  loading?: 'stripes' | boolean
+  css?: unknown
+  className?: string
+}
+
+export type ProgressBarProps = Props &
+  Omit<React.HTMLAttributes<HTMLDivElement>, keyof Props>
+
+const ProgressBar = React.forwardRef<HTMLDivElement, ProgressBarProps>(
+  (props, ref) => {
+    const {
+      // StyledComponentProps
+      css = {},
+      className = '',
+      // ComponentProps
+      loading = false,
+      label,
+      rightLabel,
+      size = 'lg',
+      barColor = '#EDEBE9',
+      color: inputColor = 'blue',
```

Comment:
question: have you written default css for this?

Reply:
fixed